barcode-network/roadsidereporting
# Fix: broken deployment

Fix broken deployment which arose due to accidentally merged merge-conflict

## Description
This PR makes the following changes:

1. Replace `class` with `className` on `src/components/Header.js`
2. Replace `class` with `className` on `src/components/Map.js`
3. Fix broken markup on `src/components/Map.js`

## Related Issue / Asana Task
N/A

## Motivation and Context
The app was not deploying

## How Has This Been Tested?
These changes can be tested as follows

1. Switch to the `fix/broken-deployment` branch.
2. Run `yarn start` in the command line
3. All tests should pass successfully

## Screenshots (if appropriate):
N/A
